README.md

```diff
@@ -487,91 +487,122 @@ gated on a condition, like tests passing.
-through 7 ahead of the actual promotion time.
-
-1) Change into the Azul project root directory: `cd azul`
-
-2) Checkout the target branch: `git checkout integration`
-
-3) This cheat sheet may differ from branch to branch. Be sure to follow the
-   cheat sheet in the README on the branch currently checked out.
-
-4) Run `git status` and make sure that your working copy is clean and the
-   branch is up-to-date.
-
-5) Merge the source branch: `git merge develop` and resolve any conflicts.
-   Conflict resolution should only be necessary if cherry-picks occured on the
-   target branch.
-
-6) The merge may have affected the README. Make sure you're looking at the
-   right version.
+1) **Initial Setup** (You can skip this step if you've deployed or promoted with
+   Gitlab at least once already)
+   
+      1) For promotion, we recommend keeping a separate clone of Azul that is never in
+         a dirty state. To create this if it doesn't yet exist run
+         ```
+         git clone git@github.com:DataBiosphere/azul.git azul.stable
+         ```
+         
+      1) Next you will need to login to our 
+         [Gitlab instance](https://gitlab.dev.explore.data.humancellatlas.org/)
+         in order to be able to push to Gitlab which automatically takes care of most
+         of the deployment process. If you haven't signed on yet, sign on with Github. You
+         will need at least `developer` permissions in order to be able to `push` to
+         Gitlab. Contact @hannes-ucsc if you have problems signing on or have
+         insufficient permissions. 
+         
+      1) [set up your ssh key](https://gitlab.dev.explore.data.humancellatlas.org/profile/keys)
+         So that you can push to Gitlab.
+         
+      1) Now that your ssh key is set up, you will need to add Gitlab as a remote. Run
+         ```
+         git remote add gitlab git@ssh.gitlab.dev.explore.data.humancellatlas.org:azul/azul.git
+         ```
+         Run
+         ```
+         git fetch gitlab
+         ```
+         to ensure that your connection is working.
+    
+1) **Prepare for Promotion** (Skip these step if you are deploying without promoting)
+   
+   1) If promoting to `staging` or `prod` you will need to prepare release notes. 
+      Make sure you do this **at least 24 hours in advance**. A link to the release
+      notes document can be found either in the #dcp-ops channel in HCA Slack or in
+      the [release guide document](https://allspark.dev.data.humancellatlas.org/dcp-ops/docs/wikis/SOP:%20Releasing%20new%20Versions%20of%20DCP%20Software)
+      1) To produce the list of changes for the DCP release notes, run 
+         ```
+         git log --pretty=oneline --topo-order --abbrev-commit
+         ```
+         All non-merge commits from the top down to the commit labeled with the
+         most recent `deployed/…` tag represent changes to be deployed and should
+         be mentioned in the release notes.
```

Comment:
The instructions should also include the removal of the tag names from the log.

Reply:
Added.